JudoPayDotNetIntegrationTests/Configuration.cs

```diff
@@ -1,12 +1,16 @@
-﻿namespace JudoPayDotNetIntegrationTests
+﻿using System.Configuration;
+using JudoPayDotNet.Enums;
+
+namespace JudoPayDotNetIntegrationTests
 {
-    static class Configuration
+    public class Configuration
     {
-        public const string Judoid = "100915867";
```

Comment:
Right, but these values haven't been moved to app.config. They've been entirely removed from the codebase. If this was intentional (that is, we no longer want these credentials to be public), we should revoke them.